vendor/github.com/vmware/photon-controller-go-sdk/SSPI/sspi.go

```diff
@@ -0,0 +1,276 @@
+// +build windows
+
+package SSPI
+
+import (
+	"fmt"
+	"strings"
+	"syscall"
+	"unsafe"
+)
+
+var (
+	secur32_dll           = syscall.NewLazyDLL("secur32.dll")
+	initSecurityInterface = secur32_dll.NewProc("InitSecurityInterfaceW")
```

Comment:
Thanks for following up. If it's a standard lib on all versions of windows we support, that's seems ok, I just wanted to check.